components/atom/backToTop/src/index.js

```diff
@@ -35,7 +36,7 @@ const AtomBackToTop = ({
       const {scrollTop, scrollHeight, clientHeight} = container
       const halfHeight = Math.floor((scrollHeight - clientHeight) / 2)
 
-      if (scrollTop > halfHeight) {
+      if (scrollTop > halfHeight || scrollTop > minHeight) {
```

Comment:
What happens if `minHeight` is `undefined`? Or the new prop is mandatory (that would be a major) or we do something in order to avoid breaking the component if no `minHeight` is passed by props.